README.md

```diff
@@ -562,6 +562,21 @@ The crontab format respects all [standard rules][cron] and has one minute
 resolution. Jobs are considered unique for most of each minute, which prevents
 duplicate jobs with multiple nodes and across node restarts.
 
+Workers which will be used in recurring jobs do not need to specify a queue.
```

Comment:
This isn't accurate. If no queue is specified then they will use the `default` queue, just like any other job.

Reply:
Ah, ok, thanks.